webpack.common.config.js

```diff
@@ -125,7 +130,7 @@ module.exports = {
     }),
     // pull common js and webpack runtime out of all bundles
     new webpack.optimize.CommonsChunkPlugin({
-      name: 'app',
```

Comment:
Yes, this was intentional. Before, `app` was the first entry point to the JS, so it contained the common JS (and webpack runtime). Now the `spam_referrals_blocker` entry point is always executed first (in the `head`), so this change pulls out common JS (which at this point is just the webpack runtime) into that entry point.